data.sql

```diff
@@ -11,3 +11,24 @@ VALUES ('Pikachu', '2021-01-07', 15.04,false, 1);
 
 INSERT INTO animals (name, date_of_birth,  weight_kg, neutered, scape_attempts)
 VALUES ('Devimon', '2017-05-12', 11,true, 5);
+-- query and update animals table lesson starts here;
+INSERT INTO animals (name, date_of_birth, weight_kg, neutered,scape_attempts)
+VALUES ('Charmander','2020-02-08',11,false,0);
+
+INSERT INTO animals (name, date_of_birth, weight_kg, neutered,scape_attempts)
+VALUES ('Plantmon','2021-11-15',5.7,true,2);
+
+INSERT INTO animals (name, date_of_birth, weight_kg, neutered,scape_attempts)
+VALUES ('Squirtle','1993-04-02',12.13,false,3);
```

Comment:
- [ Required ] I think it would be better if the values inserted in the `weight_kg` column in the `animals` table match the values provided in the project requirements. If you observe the values in the project requirements you'll notice that they are `negative` for the first three `INSERT` statements while in your code they are `positive`. Please see the screenshot below for more details on this and rectify. 

## Project requirements
![Screenshot from 2023-08-24 02-40-19](https://github.com/Diegogagan2587/vet-clinic-database/assets/16392046/3f2842f8-9f4f-431f-b662-9283e7c5e31c)